### PR TITLE
Add devise gem

### PR DESCRIPTION
**Pull request by shonendumm**
Please take a look!
Added devise gem to gemfile.

### CODE REVIEW

**Comment by shonendumm** [comment]
Thanks! 👍

File: config/environments/development.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+
+
```
Comment (gohdaniel15) [inline config/environments/development.rb]: Extra blank lines detected

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root "home#index"
```
Comment (gohdaniel15) [inline config/routes.rb]: There doesn't appear to be a `home` controller yet in this app. What would happen now if I went to that URL?

In the future, if we have this line here, it should be accompanied with a working controller and view, to ensure nothing explodes 😄

File: config/environments/development.rb (hunk 1 above)
Comment (shonendumm) [inline config/environments/development.rb]: thanks, will edit it.

File: config/routes.rb (hunk 2 above)
Comment (shonendumm) [inline config/routes.rb]: Icic, will edit it out, thanks!

**Pull request closed without merging**